packages/carbon-graphs/webpack/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+
+const path = require('path');
+const merge = require('webpack-merge');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { jsOptions, cssOptions } = require('./helpers/rules');
+
+const webpackConfig = (env, folder = '') => ({
+  mode: 'production',
+  bail: true,
+  entry: {
+    'carbon-graphs': './packages/carbon-graphs/src/js/carbon.js',
+    style: './packages/carbon-graphs/src/less/carbon.less',
+  },
+  output: {
+    path: path.resolve(__dirname, '..'),
+    filename: `./${folder}js/[name].js`,
+    library: 'Carbon',
+    libraryExport: 'default',
+    libraryTarget: 'umd',
+  },
+  resolve: {
+    modules: [process.cwd(), 'node_modules'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(less|css)$/,
+        use: [MiniCssExtractPlugin.loader, ...cssOptions(env.TYPE)],
+      },
+      {
+        test: /\.(js|jsx)$/,
+        include: [path.join(__dirname, '../src/js')],
+        use: jsOptions(env.TYPE),
+      },
+    ],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: `./${folder}css/carbon-graphs.css`,
+      chunkFilename: `./${folder}css/carbon-graphs.css`,
+    }),
+  ],
+  stats: {
+    colors: true,
+    errorDetails: true,
+    chunks: false,
+    entrypoints: false,
+    chunkModules: false,
+    chunkOrigins: false,
+    modules: false,
+    warnings: false,
+  },
+});
+
+module.exports = function (env) {
+  const custom = {};
+  if (env.TYPE === 'LIB') {
+    custom.externals = {
+      d3: 'd3',
+      c3: 'c3',
```

Comment:
is C3 a transitive dependency that we need to declare as an external here?

Reply:
From what I can gather, I believe so. This was copied as is from the old repo.